### PR TITLE
.module.css added

### PR DESCRIPTION
**Pull request by 5HW314**

### CODE REVIEW

File: backend/app.py [hunk 1]
```diff
@@ -49,4 +51,54 @@ def get_blog(id):
                 return jsonify({"error": "Blog not found"}), 404
     except Exception as e:
         return jsonify({"error": str(e)}), 500
-    
\ No newline at end of file
+    
+
+# Endpoint 3: Create New Blogs
+@app.route('/blogs/create', methods=['POST'])
```
Comment (Bhaveshshah777) [inline backend/app.py]: Please use "blogs/" endpoint only. Change it around the project

File: backend/app.py [hunk 2]
```diff
@@ -49,4 +51,54 @@ def get_blog(id):
...
+            connection.execute(query, {"title": title, "content": content, "link": link})
+            connection.commit()
+            
+        return jsonify({"message": "blog created successfully"}), 201
+    except Exception as e:
+        return jsonify({"error": str(e)}), 500
+    
+
+# Endpoint 3: Edit Blogs
+@app.route('/blogs/update/<int:id>', methods=['PUT'])
```
Comment (Bhaveshshah777) [inline backend/app.py]: Same here

**Review by Bhaveshshah777 — CHANGES_REQUESTED** [review]

File: backend/app.py [hunk 3]
```diff
@@ -49,4 +51,54 @@ def get_blog(id):
...
+            connection.execute(query, {"title": title, "content": content, "link": link})
+            connection.commit()
+            
+        return jsonify({"message": "blog created successfully"}), 201
+    except Exception as e:
+        return jsonify({"error": str(e)}), 500
+    
+
+# Endpoint 4: Edit Blogs
+@app.route('/blog/<int:id>', methods=['PUT'])
```
Comment (5HW314) [inline backend/app.py:79]: done

File: backend/app.py [hunk 4]
```diff
@@ -49,4 +51,54 @@ def get_blog(id):
                 return jsonify({"error": "Blog not found"}), 404
     except Exception as e:
         return jsonify({"error": str(e)}), 500
-    
\ No newline at end of file
+    
+
+# Endpoint 3: Create New Blogs
+@app.route('/blogs', methods=['POST'])
```
Comment (5HW314) [inline backend/app.py:57]: done

**Review by 5HW314 — COMMENTED** [review]
done

**Review by Bhaveshshah777 — CHANGES_REQUESTED** [review]
Still not done

**Review by Bhaveshshah777 — APPROVED** [review]

**Pull request merged**